specification/schemaregistry/SchemaRegistry/main.tsp

```diff
@@ -0,0 +1,282 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+@useAuth(AzureAuth)
+@service({
+  title: "Azure.SchemaRegistry",
+})
+
+// Supported operations.
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups?api-version={apiVersion}
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/$schemas/{id}?api-version={apiVersion}
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/versions/?api-version={apiVersion}
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/versions/{schemaVersion}?api-version={apiVersion}
+//  POST https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/:get-id?api-version={apiVersion}
+//  PUT https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}?api-version={apiVersion}
+@server(
+  "{fullyQualifiedNamespace}",
+  "The Schema Registry service endpoint.",
+  {
+    @doc("The Schema Registry service endpoint, for example 'my-namespace.servicebus.windows.net'.")
+    fullyQualifiedNamespace: url,
```

Comment:
I think if it's not the full endpoint then this should be typed as string?

Reply:
sorry, just updated the description, but it should include https://. If the user doesn't pass it in with that, we append it in the wrapper client before passing to the generated client.